docs/app/pages/components/components-sections/buttons/dropdowns/dropdowns.component.html

```diff
@@ -118,6 +118,8 @@
     by accessing the menu trigger directive using <code>@ViewChild(MenuTriggerDirective)</code>.
 </p>
 
+<p> When using *ngFor make sure to use the trackby function so that the menu trigger directive detects the close event.</p>
```

Comment:
Use `<code>` to mark up code in the documentation. Also I believe it should be `trackBy` (note case).

Reply:
This description is also not exactly correct. `trackBy` is required to prevent the menu from closing unexpectedly if Angular ngFor updates the items.